### ISSUE TITLE
Unexpected number of remote volumes marked as deleted

### ISSUE THREAD

**Issue by jfspan** [duplicati/duplicati#1479]
I received an error at first that there was an extra file in the backup set. So, I first tried repairing local database, then verifying the backup data. The error was repeated (and the backup wouldn't continue), so I deleted the local database and had it recreated. However, now the error is "Error while running Backup: Unexpected number of remote volumes marked as deleted. Found 0 filesets, but 12 volumes" and I haven't found a way to get around this.

Are there any other steps I can take to restore the Backup functionality? Am I doing something wrong here?

The bug report is about 43MB, so I have linked to it (GitHub won't let me upload anything larger than 10MB?).  Is there anything else I should give to help diagnose the situation?
https://onedrive.live.com/redir?resid=A60CC657F0C95379!17522&authkey=!ADBa1TlvMK-2Tyc&ithint=file%2czip

Thank you for your help and for making a backup product that I can get behind.

**Comment by jfspan** [comment]
Ok, I did more searching and found this thread #1437 that appeared similar in some of the errors, so I updated to Preview 2.0.0.90 and through deleting and repairing the database, I was able to resume backups. The repair noted that one block was listed as a longer length that it actually was on the server. 

So, the latest version appears to fix my problem. Sorry, I should have done more searching before posting (I looked for about 20 mins through other issues without finding anything).
